muy_map_conf.js

```diff
@@ -16,6 +16,15 @@ function init(config) {
   var mapMarkers = [];
   var openInfoWindow;
 
+  //If we can get location information from the browser, update the map's center point & zoom level
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(function(position) {
+      map.setCenter(new google.maps.LatLng(position.coords.latitude, position.coords.longitude));
+      map.setZoom(12);
+      console.log(position.coords.latitude, position.coords.longitude);
```

Comment:
👀

Reply:
I really need to stop working on this at 3 in the morning lol